.github/workflows/lint.yaml

```
@@ -0,0 +1,21 @@
+name: Lint
+on: [push, pull_request]
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Let's give it a name:
```suggestion
      - name: Checkout
        uses: actions/checkout@v2
```